### ISSUE TITLE
add a automated changelog and release notes

### ISSUE THREAD

**Issue by altsang** [ExpressGateway/express-gateway#188]

**Comment by IrfanBaqui** [comment]
See proposed version release process here: https://docs.google.com/a/lunchbadger.com/document/d/19yyzFS6j0hMFUcr98qcbL7Wgb86c_MLA-Cmaprb98gk/edit?usp=sharing

I propose there shouldn't be an automated changelog generator. It should be an entirely manual process where one uses the diff of git commits to separate out features in categories listed in the doc and position details as necessary.

**Comment by kevinswiber** [comment]
@IrfanBaqui 

Totally agree.  I went down the road of automated changelog once before.  It ended up not being worth it, as what the output communicates isn't exactly that great.  The output _always_ needed to be augmented significantly.

Example generated output: https://github.com/zettajs/zetta/releases/tag/v0.15.0

Hand-crafted example: https://github.com/zettajs/zetta/releases/tag/v1.0.0-beta

Huge difference.

I think the practice we started was reviewing merged PRs between releases and reporting on those.  Adding links to the PRs give a lot more context on what's going on and how decisions were made.
